src/icons/hicolor/scalable/apps/yast-cio.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The icon looks like an USB flash disk and this module does the "IO Channel management" for the S390 mainframes. And AFAIK these machines do not have USB ports at all. :wink: 

A channel is more like a specialized IO processor or device. (See more details at https://en.wikipedia.org/wiki/Channel_I/O.)